lib/customer.rb

```diff
@@ -0,0 +1,31 @@
+# create customer class
+require 'csv'
+
+class Customer
+  attr_accessor :email, :address
```

Comment:
`email` and `address` do not need to be accessors.  I suggest making them readers unless you need to give users of your class the ability to directly change the value of an instance variable.